homeassistant/components/home_connect/strings.json

```diff
@@ -21,6 +22,129 @@
       "default": "[%key:common::config_flow::create_entry::authenticated%]"
     }
   },
+  "entity": {
+    "binary_sensor": {
+      "bsh-common-status-remote_control_active": {
+        "name": "Remote control"
+      },
+      "bsh-common-status-remote_control_start_allowed": {
+        "name": "Remote start"
+      },
+      "refrigeration-common-status-door-chiller_common": {
+        "name": "Chiller door alarm"
+      },
+      "refrigeration-common-status-door-freezer": {
+        "name": "Freezer door"
+      },
+      "refrigeration-common-status-door-refrigerator": {
+        "name": "Refrigerator door"
+      }
+    },
+    "light": {
+      "bsh-common-setting-ambient_light_enabled": {
+        "name": "Ambient light"
+      },
+      "cooking-common-setting-lighting": {
+        "name": "Functional light"
+      },
+      "refrigeration-common-setting-light-external-power": {
+        "name": "External light"
+      },
+      "refrigeration-common-setting-light-internal-power": {
+        "name": "Internal light"
+      }
+    },
+    "sensor": {
+      "bsh-common-option-duration": {
+        "name": "Duration"
+      },
+      "bsh-common-option-program_progress": {
+        "name": "Program progress"
+      },
+      "bsh-common-option-remaining_program_time": {
+        "name": "Remaining program time"
+      },
+      "bsh-common-status-operation_state": {
+        "name": "Operation state",
+        "state": {
+          "bsh-common-enum_type-operation_state-inactive": "Inactive",
+          "bsh-common-enum_type-operation_state-ready": "Ready",
+          "bsh-common-enum_type-operation_state-delayed_start": "Delayed Start",
+          "bsh-common-enum_type-operation_state-run": "Run",
+          "bsh-common-enum_type-operation_state-pause": "Pause",
+          "bsh-common-enum_type-operation_state-action_required": "Action required",
+          "bsh-common-enum_type-operation_state-finished": "Finished",
+          "bsh-common-enum_type-operation_state-error": "Error",
+          "bsh-common-enum_type-operation_state-aborting": "Aborting"
```

Comment:
Because these states get annoying

Reply:
Because I ended answering to the comment from the other PR, I'm going to add the comment link here: https://github.com/home-assistant/core/pull/126157#discussion_r1768269261